src/en/getting-started.md

```diff
@@ -195,13 +199,12 @@ details  {
    Juju via a snap package. See the [install][install] for more information.
 
 
-^# From the Ubuntu PPA
+^# For Ubuntu
```

Comment:
I think i prefer it as it was - this makes it seem like this is the part for Ubuntu users, when in reality the approved/default method is installing the snap